client/src/components/App/Content/InvestigationForm/ConversationScript/Scripts/ClinicalDetails.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Typography } from '@material-ui/core'
+
+const PersonalDeatils = (props: Props) => {
```

Comment:
you can remove the unused props here and at the bottom of the component

Reply:
About that, I was thinking that leaving the props value in will make it easier to use in the furture in case we'll want to add props.

in that note - I don't really know what's better in this case - well talk about it in person on sunday :)